storage/scanner.go

```diff
@@ -72,13 +70,12 @@ type storeStats struct {
 // interval).  Each range is tested for inclusion in a sequence of
 // prioritized range queues.
 type rangeScanner struct {
-	targetInterval time.Duration  // Target duration interval for scan loop
-	maxIdleTime    time.Duration  // Max idle time for scan loop
-	ranges         rangeSet       // Ranges to be scanned
-	queues         []rangeQueue   // Range queues managed by this scanner
-	removed        chan *Range    // Ranges to remove from queues
-	stats          unsafe.Pointer // Latest store stats object; updated atomically
-	scanFn         func()         // Function called at each complete scan iteration
+	targetInterval time.Duration // Target duration interval for scan loop
+	maxIdleTime    time.Duration // Max idle time for scan loop
+	ranges         rangeSet      // Ranges to be scanned
+	queues         []rangeQueue  // Range queues managed by this scanner
+	removed        chan *Range   // Ranges to remove from queues
+	scanFn         func()        // Function called at each complete scan iteration
```

Comment:
Does anything else use the scanFn?  I think we might be able to remove that as well.